### ISSUE TITLE
I can't quit the File Options dialog with ESC

### ISSUE THREAD

**Issue by softwarenerd** [posit-dev/positron#15922]
#### Positron and OS details:

Positron Version: 2026.08.2 build 4
Code - OSS Version: 1.124.0
Commit: 747dd536c5913adad71a568a57679bb2c5e5cd71
Date: 2026-08-20T19:13:05Z
Electron: 42.2.0
Chromium: 148.0.7778.97
Node.js: 24.15.0
V8: 14.8.178.14-electron.0
OS: Darwin arm64 25.6.0

#### Session details:

N/A

## Describe the issue:

It should be possible to quit the File Options dialog with ESC.

<p>
<img width="1782" height="1035" alt="Image" src="https://github.com/user-attachments/assets/558982c9-bdeb-4d77-8d44-9d3fb39a78f1" />
<p>

It doesn't work.

## Expected or desired behavior:

File Options dialog should close.

**Comment by juliasilge** [comment]
Closed by #16222
